src/table/internal.tsx

```diff
@@ -597,15 +598,17 @@ const InternalTable = React.forwardRef(
                                       onFocusDown: moveFocusDown,
                                       onFocusUp: moveFocusUp,
                                       rowIndex,
-                                      itemKey: `${getTableItemKey(row.item)}`,
+                                      itemKey: rowId,
                                     }}
                                     verticalAlign={cellVerticalAlign}
                                   />
                                 )}
 
                                 {visibleColumnDefinitions.map((column, colIndex) => {
-                                  const isEditing = cellEditing.checkEditing({ rowIndex, colIndex });
-                                  const successfulEdit = cellEditing.checkLastSuccessfulEdit({ rowIndex, colIndex });
+                                  const colId = `${getColumnKey(column, colIndex)}`;
```

Comment:
Unnecessary string interpolation, this could be just
```
const colId = getColumnKey(column, colIndex);
```

Reply:
It is needed because getColumnKey() returns string or number. However, I made the cellId.col and cellId.row to only accept strings for better predictability.